codegens/nodejs-native/lib/request.js

```diff
@@ -15,19 +15,48 @@ var self;
  */
 function makeSnippet (request, indentString, options) {
   var nativeModule = (request.url.protocol === 'http' ? 'http' : 'https'),
-    snippet = `var ${nativeModule} = require('${nativeModule}');\n`,
+    snippet,
     optionsArray = [],
     postData = '';
-
+  if (options.ES6_enabled) {
+    snippet = 'const ';
+  }
+  else {
+    snippet = 'var ';
+  }
+  snippet += `${nativeModule} = require('${nativeModule}');\n`;
   if (options.followRedirect) {
-    snippet = `var ${nativeModule} = require('follow-redirects').${nativeModule};\n`;
+    if (options.ES6_enabled) {
+      snippet = 'const ';
+    }
+    else {
+      snippet = 'var ';
+    }
+    snippet += `${nativeModule} = require('follow-redirects').${nativeModule};\n`;
+  }
```

Comment:
Can you refactor this flow, such that we only test for ES6_enabled only once?
Something like below
```
if (options.ES6_enabled) {
  snippet = 'const ';
}
else {
  snippet = 'var ';
}
if (options.followRedirect) {
 require('follow-redirects).nativeModule;
}
else {
  require('nativeModule');
}
```